pkgs/development/beam-modules/build-mix.nix

```diff
@@ -60,15 +80,16 @@ let
 
       # Some packages like db_connection will use _build/shared instead of
       # honoring the $MIX_ENV variable.
-      for reldir in _build/{$MIX_ENV,shared}/lib/${name}/{src,ebin,priv,include} ; do
+      for reldir in _build/{$MIX_ENV,shared}/lib/${name}/{src,ebin,priv,include};
+      do
```

Comment:
```suggestion
      for reldir in _build/{$MIX_ENV,shared}/lib/${name}/{src,ebin,priv,include}; do
```

Reply:
I messed up this commit. This should definitely not have been there